test/users.spec.ts

```diff
@@ -86,6 +86,16 @@ describe('Users', function () {
     expect(numUsers).to.equal(1);
   });
 
+  it.skip('gets the profile', async () => {
```

Comment:
Skipping these tests for now as these require a feature flag to be enabled for Keycloak.